package.json

```diff
@@ -75,6 +75,7 @@
         "split.js": "1.6.2",
         "ts-cacheable": "1.0.2",
         "tslib": "2.0.3",
+        "weak-napi": "^2.0.2",
```

Comment:
```suggestion
        "weak-napi": "2.0.2",
```

Reply:
and please put it into devDependencies

Reply:
Done in 089e7020eedb5ed6cc98a7425f377195048d72ee